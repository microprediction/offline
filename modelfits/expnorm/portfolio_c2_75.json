{"g1": 0.033084492376769976, "g2": 13.06708706729452, "logK": -1.152886783688174, "loc": 0.08182666445307762, "logScale": 0.8046802333957237}