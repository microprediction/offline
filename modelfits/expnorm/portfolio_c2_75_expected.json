{"g1": 0.6171725407619187, "g2": 14.292531675940136, "logK": 0.9978226953907283, "loc": -0.07235135561726419, "logScale": -2.3151431773953304}